kansha/comment/comp.py

```diff
@@ -116,12 +116,13 @@ def flow_elements(self):
     def _create_comment_component(self, data_comment):
         return component.Component(Comment(data_comment)).on_answer(self.delete)
 
-    def add(self, v):
+    def add(self, v, comp):
         """Add a new comment to the card
 
         In:
             - ``v`` -- the comment content
         """
+        # TODO: get rid of comp from add and delete, and isolate comp operations in a function handle_event or something.
         security.check_permissions('comment', self.parent)
         if v is None:
```

Comment:
Are we really calling this `add` method with `None` instead of a text somewhere?